tests/virtual_machines_test.go

```diff
@@ -270,6 +275,42 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					})
 				})
 			})
+
+			Context("and creating two VMs with same mac, one on opt-out namespace", func() {
```

Comment:
This test is running under `Context("When Running with default opt-mode configuration"` 
right now that means it's opt-out, but we can't rely on that. 
Since your tests is running assuming that we're running in opt-out, you need to move it under `Context("When all the VMs are serviced by default (opt-out mode)"`